.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - os: Linux
+          - os: heavy
```

Comment:
This should be changed everywhere not only in one place. I think it would be better to create another PR for that to not mix different changes in one PR.

Reply:
I changed it in my PR #1078 for all workflows so don't worry about it.